packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -623,6 +697,10 @@ export default class ConversationService {
     });
   }
 
+  public getImage(data: RemoteData): Promise<Buffer> {
```

Comment:
We can remove this legacy method and run a beta release because with my PR (#929) we will need a major release anyway.